specification/digitaltwins/data-plane/Microsoft.DigitalTwins/preview/2021-06-30-preview/digitaltwins.json

```diff
@@ -0,0 +1,1621 @@
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Error"
+          },
+          "readOnly": true
+        },
+        "innererror": {
+          "description": "An object containing more specific information than the current object about the error.",
+          "$ref": "#/definitions/InnerError"
+        }
+      }
+    },
+    "InnerError": {
+      "description": "A more specific error description than was provided by the containing error.",
+      "properties": {
+        "code": {
+          "description": "A more specific error code than was provided by the containing error.",
+          "type": "string"
+        },
+        "innererror": {
+          "description": "An object containing more specific information than the current object about the error.",
+          "$ref": "#/definitions/InnerError"
+        }
+      }
+    }
+  },
+  "securityDefinitions": {
+    "oauth2": {
+      "flow": "implicit",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize?resource=0b07f429-9f4b-4714-9392-cc5e8e80c8b0",
+      "type": "oauth2"
+    }
+  },
+  "parameters": {
+    "eventRouteId": {
+      "name": "id",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "The id for an event route. The id is unique within event routes and case sensitive.",
+      "x-ms-parameter-location": "method"
+    },
+    "includeModelDefinition": {
+      "name": "includeModelDefinition",
+      "description": "When true the model definition will be returned as part of the result.",
+      "in": "query",
+      "required": false,
+      "type": "boolean",
+      "default": false,
+      "x-ms-parameter-location": "method"
+    },
+    "digitalTwinId": {
+      "name": "id",
+      "in": "path",
+      "description": "The id of the digital twin. The id is unique within the service and case sensitive.",
+      "type": "string",
+      "required": true,
+      "x-ms-parameter-location": "method"
+    },
+    "modelId": {
+      "name": "id",
+      "in": "path",
+      "description": "The id for the model. The id is globally unique and case sensitive.",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "method"
+    },
+    "relationshipId": {
+      "name": "relationshipId",
+      "description": "The id of the relationship. The id is unique within the digital twin and case sensitive.",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "method"
+    },
+    "componentPath": {
+      "name": "componentPath",
+      "in": "path",
+      "description": "The name of the DTDL component.",
+      "type": "string",
+      "required": true,
+      "x-ms-parameter-location": "method"
+    },
+    "api-version": {
+      "name": "api-version",
+      "in": "query",
+      "description": "The requested API version.",
+      "required": true,
+      "type": "string"
```

Comment:
This is the only difference I see when comparing to the previous API version.  Is the intention to make this configurable?

Reply:
This was done since updating the enum value to the new API version was reported as a breaking change. I saw that other providers use this configurable form, so I was thinking that maybe this is preferred?

Reply:
Doing it this way will make the API version a parameter which I'm guessing isn't what you want.  If the linter is complaining we can get that fixed as it shouldn't flag this (it's the whole point of creating a new API version).

Reply:
Makes sense - I've added it back in. This will cause one of the breaking change checks to fail, but if you can override that, that would be great.